### ISSUE TITLE
Settings page

### ISSUE THREAD

**Issue by digitaltom** [digitaltom/photobooth#21]
A settings page where the config can be changed and for example `gphoto2 --storage-info` is shown.

**Comment by ChrisBr** [comment]
But how would you access the settings page without the IP?

**Comment by digitaltom** [comment]
@ChrisBr yes that's the problem... -> https://github.com/digitaltom/photobooth/issues/29 
Any ideas? It would be cool to announce to something like dyndns...
